### DISCUSSION TITLE
When to release TRAPI 1.3

### DISCUSSION THREAD

**Discussion by edeutsch** [NCATSTranslator/ReasonerAPI#311] [General]
As we discussed at last week's call, it is an open question as to when we should release TRAPI 1.3 with the features that we have finished. The options were:

A. Release ASAP in its current state and get everyone to implement and roll out in January on a fairly quick schedule so that everyone is on TRAPI 1.3 by the Feb Relay
B. Release ASAP in its current state but don't require everyone to implement it in January. Some will some won't and that's okay.
C. Stay with TRAPI 1.2 for the Feb Relay, and release TRAPI 1.3 after the Relay sometime, perhaps with additional work

To vote, please select ONE emoji on this post:
A. Rocket ship! Let's carpe diem and force progress!
B. Thumbs up! Let's checkpoint what we have as a new release but be relaxed about implementation
C. Thumbs down! The proposed features just aren't crucial for the Relay, so let's not cause chaos in January

**Comment by brettasmi** [comment]
I should have voted after the previous meeting because now I'm a bit forgetful as to what outstanding issues might be included in a 1.3 release if we don't checkpoint/release now. Maybe a quick refresher tomorrow would be helpful to get back in the right headspace for voting.
